src/index.html

```diff
@@ -6,7 +6,269 @@
+          <img
+            class="section__photo"
+            src="./images/imac.jpeg"
+            alt="monitor"
+          >
+          <div class="section__header">
+            <span class="section__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+            <span class="section__code">Код товара: 195434</span>
+          </div>
+          <div class="section__inner">
+            <div class="section__stars">
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+            </div>
+            <div class="section__feedback">Отзывов: 5</div>
+          </div>
+          <div class="section__sell">
+            <span class="section__desription">Цена:</span>
+            <span class="section__price">69 999 грн</span>
+          </div>
+          <a class="section__bottom" data-qa="card-hover" >купить</a>
+        </div>
+        <div class="wrap__section section">
+          <img
+            class="section__photo"
+            src="./images/imac.jpeg"
+            alt="monitor"
+          >
+          <div class="section__header">
+            <span class="section__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+            <span class="section__code">Код товара: 195434</span>
+          </div>
+          <div class="section__inner">
+            <div class="section__stars">
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+            </div>
+            <div class="section__feedback">Отзывов: 5</div>
+          </div>
+          <div class="section__sell">
+            <span class="section__desription">Цена:</span>
+            <span class="section__price">69 999 грн</span>
+          </div>
+          <a class="section__bottom">купить</a>
+        </div>
+        <div class="wrap__section section">
+          <img
+            class="section__photo"
+            src="./images/imac.jpeg"
+            alt="monitor"
+          >
+          <div class="section__header">
+            <span class="section__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+            <span class="section__code">Код товара: 195434</span>
+          </div>
+          <div class="section__inner">
+            <div class="section__stars">
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+              <div class="section__stars-item"></div>
+            </div>
```

Comment:
вот так будет правлельнее
```suggestion
            <div class="section__stars starts">
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
            </div>
```
а так правильно 
```suggestion
            <div class="section__stars starts starts--4">
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
            </div>
```